### ISSUE TITLE
two examples do not compose

### ISSUE THREAD

**Issue by alanedelman** [JuliaParallel/Elemental.jl#65]
on the readme
the first example and the second example do not compose
a new user would naturally try both and get frustrated

perhaps the example that nominally states `without MPI` has a uniprocessor
MPI under the hood causing interference?

but still this is frustrating as a user experience

```
julia> using LinearAlgebra, Elemental

julia> A = Elemental.Matrix(Float64)
0×0 Elemental.Matrix{Float64}

julia> using MPI, MPIClusterManagers, Distributed

julia> man = MPIManager(np = 4);
ERROR: AssertionError: MPI.Initialized() == mgr.launched
Stacktrace:
 [1] MPIManager(; np::Int64, launch_timeout::Float64, mode::TransportMode, master_tcp_interface::String) at /Users/edelman/.julia/packages/MPIClusterManagers/0ZYYQ/src/mpimanager.jl:64
 [2] top-level scope at REPL[4]:1
```

**Comment by andreasnoack** [comment]
I can't reproduce this. Is it possible that you've called `MPI.Init()` in your session?

**Comment by alanedelman** [comment]
nope, the entire session is as typed in 1.4.2

**Comment by andreasnoack** [comment]
Which versions of `Elemental`, `MPI` and `MPIClusterManagers` are you using?

**Comment by ViralBShah** [comment]
Tagged versions (currently). BTW, I had the same issue.

**Comment by ViralBShah** [comment]
I do believe this used to work before the recent round of tagging new versions of packages.

**Comment by andreasnoack** [comment]
Yeah. I was on a dev branch that didn't have the `_jll` stuff. After freeing, I'm getting
```julia
julia> using MPI

julia> MPI.Initialized()
false

julia> using Elemental

julia> MPI.Initialized()
true
```
which used to be false so for some reason we are now automatically initializing `MPI` when loading `Elemental`. I'm wondering if this might be because `Elemental` and `MPI` are now using the same MPI "instance" where they didn't use to.

Oh, the joy of MPI. Maybe we can add a comment that each of the examples assumes a separate session.

**Comment by ViralBShah** [comment]
Maybe just do one intialization?

**Comment by andreasnoack** [comment]
Part of the idea with the first example is that you don't have to setup an MPIClusterManager if you don't run distributed. However, this is needed for the second example.

**Comment by ViralBShah** [comment]
Is the first example single core?

**Comment by andreasnoack** [comment]
Yes. `Elemental.Matrix(Float64)` is a non-distributed array type.

**Comment by ViralBShah** [comment]
It would be nice to lead with a parallel example. I might even suggest just dropping that one. I already have LAPACK and multi-threaded blas...

**Comment by alanedelman** [comment]
i would drop the single processor example
